projects/ks89/angular-modal-gallery/src/lib/components/previews/previews.component.spec.ts

```diff
@@ -738,7 +738,15 @@ describe('PreviewsComponent', () => {
 
         previews = element.queryAll(By.css('img'));
 
-        previews[1].nativeElement.click();
+        // previews[1].nativeElement.click();
+        comp.ngOnChanges({
```

Comment:
Addresses: 1. clicking the right image arrow always shifts the displayed previews, even in cases where it shouldn't, for instance for nbPreviews>=3 and current=0